User blocking implementation (#268)
desmos-labs/desmos: CHANGELOG.md

# Version 0.12.0

> - Implemented the possibility to block/unblock a specific user `address` (#169)

Reviewer: ```suggestion
- Implemented the possibility of (un)blocking a specific user (#169)
```
